src/Form/CopyDocumentForm.php

```diff
@@ -27,10 +32,11 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder->add('cases', EntityType::class, [
             'choices' => $suitableCases,
             'class' => get_class($case),
-            'label' => 'Copy to',
+            'label' => $this->translator->trans('Copy to', [], 'documents'),
             'multiple' => true,
             'attr' => [
-                'size' => 10,
+                'class' => 'select2',
+                'data-placeholder' => $this->translator->trans('Click to select', [], 'documents'),
```

Comment:
“Click to select” is a somewhat unorthodox placeholder text. “Select cases” will make more sense.

Reply:
Reworded.